indico/web/client/js/custom_elements/ind_share_widget.module.scss

```diff
@@ -0,0 +1,74 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@use 'base/palette' as *;
+
+.share-widget {
+  cursor: pointer;
+  filter: opacity(0.6);
+
+  &:hover {
+    filter: none;
+  }
+}
+
+.show {
+  filter: none;
+}
+
+.header-text {
+  font-size: 1.4em !important;
+  margin: -2.8px 100px 8.5px !important;
```

Comment:
Not a big fan of the hard-coded font sizes and margins. Would it be possible to use some existing SUI options/components to get the same look?

Reply:
I did initially try to do some stuff with the existing SUI options/components, but I found that trying to center this large heading using the stuff in SUI didn't work (hence why I have hardcoded the `margin`). As for the `font-size`; the available sizes in SUI just didn't really cut it for me, and made everything else look a bit weird...

But I will give this another try.

Reply:
After playing around with this, I still cannot use SUI to do this (specifically the [`textAlign` property](https://react.semantic-ui.com/elements/header/#variations-text-alignment)). Although, doing this myself did lead to a somewhat similar outcome:

```html
<div styleName="header-text">
   <Icon name="share alternate" styleName="icon" />
   <Translate as="strong">Share this page</Translate>
</div>
```

```scss
.header-text {
  font-size: 1.6em;
  text-align: center;
  margin: 5px 0 5px 0;

  .icon {
    margin-right: 5px;
  }
}
```
![image](https://github.com/indico/indico/assets/7736654/8ea449a9-dd37-47b3-ba86-0eb9a0d0dcd1)

I'm not sure wether this might be an 'improvement' though... any thoughts?